application-services/custom/camera-management/README.md

```diff
@@ -215,13 +207,15 @@ cd edgex-examples/application-services/custom/camera-management
 ```
 
 ```shell
-# Build the app. 
-make build-app
+# Build the docker image. 
+make docker
 
-# Run the app.
-make run-app
+# Start the docker compose services in the background for both EVAM and Camera Management App
+docker compose up -d
```

Comment:
Since all other commands are make commands it would be nice to also have docker compose ones under make.

Reply:
normally I would agree, but in this case I think it is fine to keep as docker compose, as the user has more control over it